src/Extensions/Logging/Internal/OmexLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Logging.Replayable;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLogger : ILogger
+	{
+		public OmexLogger(ILogEventSender logsEventSource, IExternalScopeProvider externalScopeProvider, string categoryName)
```

Comment:
Validate

Reply:
This one internal  and would be created only inside `OmexLoggerProvider` that is getting types from DI, I don't think there is a need in validation